.github/workflows/build.yaml

```
@@ -29,6 +29,11 @@ on:
         required: true
         type: boolean
         default: false
+      lsp4ijPrNumber:
+        description: 'If you checked this, please make sure to enter only the LSP4IJ PR number below'
```

```suggestion
        description: 'Run build with an LSP4IJ PR. Make sure to enter the LSP4IJ PR number in the \"Reference/branch/PR for LSP4IJ checkout\" input box.'
```
*Note: I'm not actually sure if the `\"` is the correct syntax for including quotes.

---

Thanks @TrevCraw for the suggestion, I did the change.